internal/backends/qt/qt_window.rs

```diff
@@ -1516,6 +1516,51 @@ impl WindowAdapter for QtWindow {
     fn window(&self) -> &i_slint_core::api::Window {
         &self.window
     }
+
+    fn renderer(&self) -> &dyn Renderer {
```

Comment:
unfortunately, even if Renderer is sealed and not re-exported by the public crate, one will still be able to call function on them in their code.

```
window_adapter.renderer().text_size(Default::default(), "abc", Default::default(), Default::default())
```